### PR TITLE
Ravens matrices

### PR DESCRIPTION
**Pull request by katelynbottcher**
https://github.com/Watts-Lab/task-mapping/issues/31

### CODE REVIEW

File: Ravens Matrices.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+## Instructions
+> Participants are asked to look for a pattern and determine ‘what comes next’. Participants were asked to find the missing element (from options
+a to f). 
+
+# Criteria
+>  Points are given to each item where the next element in the pattern is determined correctly.
+
+## Performance calculation
+> No incentives were mentioned
```
Comment (sarikasu) [inline Ravens Matrices.md:36]: I think intelligence (IQ) was measured based on the correct number of elements in a pattern were determined

Comment (sarikasu) [inline Ravens Matrices.md:36]: I did find an incentives section: 
Participants who completed the study were paid a total of $100: $10 for completing the individual tests; $30 for lab visit 1; and $60 for lab visit 2. We elected not to explicitly pay groups for better performance, relying instead on
intrinsic motivation through priming.

File: Ravens Matrices.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# Summary
+> The series of tasks in this article worked to identify individual contributions to group performance to determine "team players." The Ravens Task in specific measured pattern recognition and spatial reasoning. This contributes to the study of IQ or liquid intelligence. 
+
+# References
+> Main paper: https://www.nber.org/system/files/working_papers/w27071/w27071.pdf
+> > Supporting paper: Raven, J. (2003). Raven’s Advanced Progressive Matrices, Test Books 1 and 2: Pearson Assessment.
+> > > CANNOT FIND ACCESS TO THIS PAPER
+
```
Comment (sarikasu) [inline Ravens Matrices.md:8]: I think they are actually test practice booklets, I think I found them on the Pearson site: https://www.pearsonassessments.com/store/usassessments/en/Store/Professional-Assessments/Cognition-%26-Neuro/Non-Verbal-Ability/Raven%E2%80%99s-Advanced-Progressive-Matrices/p/100000414.html

Comment (katelynbottcher) [reply Ravens Matrices.md:8]: Were you able to access them for free?

File: Ravens Matrices.md (hunk 1 above)
Comment (katelynbottcher) [reply Ravens Matrices.md]: good find thank you!

File: Ravens Matrices.md (hunk 2 above)
Comment (sarikasu) [reply Ravens Matrices.md:8]: No I think they need to be purchased

**Review by sarikasu — APPROVED** [review]

**Pull request merged**